### ISSUE TITLE
Install next/core-web-vitals package

### ISSUE THREAD

**Issue by Abhi942001** [super-linter/super-linter#4964]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I have integrated some unit tests in a next.js project.
I am getting this error ESLint couldn't find the config "next/core-web-vitals" to extend from. Please check that the name of the config is correct.
.eslintrc.json is this 
`{
  "extends": ["prettier", "next/core-web-vitals"],
  "root": true,
  "env": {
    "jest": true
  },
  "rules": {
    "@next/next/no-img-element": "off",
    "semi": ["error", "never"],
    "comma-dangle": ["error", "never"]
  }
}
`

### Expected Behavior

It should not show this error.

### Super-Linter version

```markdown
v5.7.2
```

### Relevant log output

```shell
File:[src/components/sections/Hero/__tests__/hero.test.js]
ERROR! Found errors in [eslint] linter!
ERROR:[
Oops! Something went wrong! :(

ESLint: 7.3.1

ESLint couldn't find the config "next/core-web-vitals" to extend from. Please check that the name of the config is correct.

The config "next/core-web-vitals" was referenced from the config file in "/action/lib/.automation/.eslintrc.json".

If you still have problems, please stop by https://eslint.org/chat to chat with the team.]
```

### Steps To Reproduce

Running the super linter normally.

**Comment by ferrarimarco** [comment]
Hi @Abhi942001 

I think this is because `next/core-web-vitals` is not currently installed in the super-linter image. That's why I'm marking this as a feature request and not as as a bug.

**Comment by Abhi942001** [comment]
@ferrarimarco let me know when can you release the new feature

**Comment by ferrarimarco** [comment]
Hi. We don't have an ETA for this. Feel free to contribute with a PR. I'll review it :)
